.github/workflows/github-actions-ci.yml

```diff
@@ -23,7 +23,6 @@ jobs:
         run: |
           source /usr/share/miniconda/etc/profile.d/conda.sh
           conda activate geo_deep_env
-          unzip ./data/massachusetts_buildings.zip -d ./data
```

Comment:
are we not testing on that dataset anymore?

Reply:
This PR was built on top of PR #264 . This modification belongs to the other PR. The test data is simply unzipped by default. I've started working on #257 and all test data will be this way (ie saved as is, not zipped)